packages/synapse-react-client/src/components/EntityHeaderTable/DebouncedInput.tsx

```diff
@@ -16,20 +16,21 @@ export function DebouncedInput({
   options: string[]
   delay?: number
 } & Pick<
-  React.InputHTMLAttributes<TextFieldProps>,
-  'type' | 'min' | 'max' | 'value' | 'placeholder' | 'className' | 'list'
```

Comment:
I wanted to add `label`. Keeping `'min' | 'max' | 'list'` caused type errors. I am not sure if those options worked, but if we need them options we can revisit--we don't seem to be using them now